clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -140,6 +140,8 @@ export default async function handler(
     ? experience.experience_config?.component === ComponentType.TCF_OVERLAY
     : environment.settings.IS_FORCED_TCF;
 
+  const gppEnabled = !!experience?.gpp_settings?.enabled;
```

Comment:
We have another ticket to ensure this var isn't tied to the experience, right?

Reply:
we have a ticket to force gpp even if it's not enabled. That's the next PR I'll give you ;)